model/relabel/relabel.go

```diff
@@ -192,24 +192,31 @@ func (re Regexp) String() string {
 // are applied in order of input.
 // If a label set is dropped, nil is returned.
 // May return the input labelSet modified.
```

Comment:
Is this comment ever true? I can't see where relabel.Process ever modifies the input label set.

Reply:
I'm sure you're right about the current code, but it seems a useful freedom to reserve so I wouldn't call the comment "stale".